src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -163,7 +173,10 @@ export class CalcitePanel {
     return canDisplayHeader ? (
       <header class={CSS.header}>
         {headerLeadingContentNode}
-        {headerContentNode}
+        <div class={CSS.headerMiddleContent}>
+          {headerContentNode}
+          {headerSummaryNode}
```

Comment:
@asangma It seems odd that there is a Content and a Summary slot. Its not obvious which goes above the other. Maybe we just need another component for `calcite-panel-header` or `calcite-flow-item-header` where a heading/summary slots could exist?  Then a user can choose to put that inside or not. Otherwise, we're requiring a heading/summary only in the middle. If that is an OK requirement, maybe we just rename the slot to heading?